docs/course_materials/README.md

```diff
@@ -0,0 +1,17 @@
+How to update/add exercises to course slides
+============================================
+
+1. Create an `.md` exercise file. Please follow the styling conventions.  
+
+2. Add a new section in the corresponding `.html` using the following:  
+```html
+<section data-markdown="relative/path/class_exercise_number.md" data-separator="^\n\n\n" data-separator-vertical="^\n\n">
+</section>
+```
+   - Example path: `"exercises/01_Linux/class_exercise_2.md"`
+   - Make sure not to use the abriviation 'exe', the styling of the slides won't work propperly.  

```

Comment:
abriviation -> abbreviation
propperly -> properly

Reply:
Fixed it and added an option to test by using `xdg-open`.